### PR TITLE
Refine test for ci

### PR DESCRIPTION
**Pull request by king41409**
refine test for ci

### CODE REVIEW

File: src/main/java/com/aliyun/oss/model/ImageProcess.java [hunk 1]
```diff
@@ -87,17 +87,17 @@ public void setSupportAtStyle(Boolean supportAtStyle) {
         this.supportAtStyle = supportAtStyle;
     }
 
-    // Img表示设置的样式分隔符，只有Img能用；Both表示oss也能用Img的样式分隔符
+    // Img said setting the style of the separator, can only Img; Both said the oss can Img style separator
```
Comment (ljbreak2008) [inline src/main/java/com/aliyun/oss/model/ImageProcess.java]: 确认一下真实含义

File: src/main/java/com/aliyun/oss/model/ImageProcess.java [hunk 2]
```diff
@@ -87,17 +87,17 @@ public void setSupportAtStyle(Boolean supportAtStyle) {
...
     private String compliedHost;
-    // 是否开启原图保护
+    // Whether to open the original protection
```
Comment (ljbreak2008) [inline src/main/java/com/aliyun/oss/model/ImageProcess.java]: Whether to open the original image protection

File: src/main/java/com/aliyun/oss/model/ImageProcess.java [hunk 3]
```diff
@@ -87,17 +87,17 @@ public void setSupportAtStyle(Boolean supportAtStyle) {
...
     private Boolean sourceFileProtect;
-    // 原图保护的后缀，*表示所有
+    // All original protection suffix, * said
```
Comment (ljbreak2008) [inline src/main/java/com/aliyun/oss/model/ImageProcess.java]: the original protection suffix,* is all suffix

File: src/main/java/com/aliyun/oss/model/ImageProcess.java [hunk 4]
```diff
@@ -87,17 +87,17 @@ public void setSupportAtStyle(Boolean supportAtStyle) {
...
     private String sourceFileProtectSuffix;
-    // 自定义样式分隔符
+    // Custom delimiter
```
Comment (ljbreak2008) [inline src/main/java/com/aliyun/oss/model/ImageProcess.java]: Custom style delimiter

File: src/main/java/com/aliyun/oss/model/ImageProcess.java [hunk 5]
```diff
@@ -87,17 +87,17 @@ public void setSupportAtStyle(Boolean supportAtStyle) {
...
     private String styleDelimiters;
-    // 图片服务的版本目前是2，只能读取不能设置
+    // Photo service version is 2, can only read cannot be set
```
Comment (ljbreak2008) [inline src/main/java/com/aliyun/oss/model/ImageProcess.java]: Photo service version is 2, you can only read but can not be set

File: src/main/java/com/aliyun/oss/model/ImageProcess.java [hunk 6]
```diff
@@ -87,17 +87,17 @@ public void setSupportAtStyle(Boolean supportAtStyle) {
...
     private Integer version;
-    // 用户是否能够通过OSS域名使用老版图片处理接口，@格式。默认Disabled
+    // Whether the user can through the OSS domain using the original image processing interface, @ format.Disabled by default
```
Comment (ljbreak2008) [inline src/main/java/com/aliyun/oss/model/ImageProcess.java]: Whether the user can use the old image processing interface through the OSS domain ,@ format,Disabled by default.

File: src/main/java/com/aliyun/oss/model/ListBucketsRequest.java [hunk 7]
```diff
@@ -165,10 +165,10 @@ public ListBucketsRequest withMaxKeys(Integer maxKeys) {
     }
 
     /**
-     * Sets the bid。
+     * Sets the bid.
```
Comment (ljbreak2008) [inline src/main/java/com/aliyun/oss/model/ListBucketsRequest.java]: Set the bid.

File: src/main/java/com/aliyun/oss/model/ListBucketsRequest.java [hunk 8]
```diff
@@ -165,10 +165,10 @@ public ListBucketsRequest withMaxKeys(Integer maxKeys) {
...
      * 
      * @param bid
-     *            bid。
+     *            bid.
```
Comment (ljbreak2008) [inline src/main/java/com/aliyun/oss/model/ListBucketsRequest.java]: 左对齐

File: src/main/java/com/aliyun/oss/common/auth/CredentialsFetcher.java [hunk 9]
```diff
@@ -30,46 +30,46 @@
...
+     * Construction license server URL
      * 
      * @return the url of authorization server
      * @throws ClientException
      */
     public URL buildUrl() throws ClientException;
 
     /**
-     * 发送HTTP请求到授权服务器
+     * Sends HTTP request to the license server
```
[the remark this answers is not in the document]
Comment (king41409) [reply src/main/java/com/aliyun/oss/common/auth/CredentialsFetcher.java]: OK

File: src/main/java/com/aliyun/oss/common/auth/CredentialsFetcher.java [hunk 10]
```diff
@@ -30,46 +30,46 @@
...
-     *            HTTP请求
+     *          
      * @return http response
      * @throws IOException
      */
     public HttpResponse send(HttpRequest request) throws IOException;
 
     /**
-     * 解析授权服务器返回的授权信息，解析为Credentials
+     * Authorization information from server, parse for Credentials
```
[the remark this answers is not in the document]
Comment (king41409) [reply src/main/java/com/aliyun/oss/common/auth/CredentialsFetcher.java]: OK

File: src/main/java/com/aliyun/oss/common/auth/CredentialsFetcher.java [hunk 11]
```diff
@@ -30,46 +30,46 @@
...
      * 
      * @param response
-     *            授权服务器返回的授权信息
+     *                license server return authorization information
```
[the remark this answers is not in the document]
Comment (king41409) [reply src/main/java/com/aliyun/oss/common/auth/CredentialsFetcher.java]: OK

File: src/main/java/com/aliyun/oss/common/auth/CredentialsFetcher.java [hunk 12]
```diff
@@ -30,46 +30,46 @@
...
      */
     public Credentials fetch() throws ClientException;
 
     /**
-     * 从授权服务器获取授权
+     * Get authorization from the authorization server
      * 
      * @param retryTimes
-     *            失败重试此时
+     *                  failure retry time
```
[the remark this answers is not in the document]
Comment (king41409) [reply src/main/java/com/aliyun/oss/common/auth/CredentialsFetcher.java]: OK

File: src/main/java/com/aliyun/oss/model/ImageProcess.java (hunk 4 above)
Comment (king41409) [reply src/main/java/com/aliyun/oss/model/ImageProcess.java]: OK

File: src/main/java/com/aliyun/oss/model/ImageProcess.java (hunk 5 above)
Comment (king41409) [reply src/main/java/com/aliyun/oss/model/ImageProcess.java]: OK

File: src/main/java/com/aliyun/oss/model/ImageProcess.java (hunk 6 above)
Comment (king41409) [reply src/main/java/com/aliyun/oss/model/ImageProcess.java]: OK

File: src/main/java/com/aliyun/oss/model/ListBucketsRequest.java (hunk 7 above)
Comment (king41409) [reply src/main/java/com/aliyun/oss/model/ListBucketsRequest.java]: OK

File: src/main/java/com/aliyun/oss/model/ListBucketsRequest.java (hunk 8 above)
Comment (king41409) [reply src/main/java/com/aliyun/oss/model/ListBucketsRequest.java]: OK

**Comment by huiguangjun** [comment]
和主版本差异很大，不需要合入。

**Pull request closed without merging**